### PR TITLE
update rules of analysis_options.yaml. and solved all problems

### PR DESCRIPTION
**Pull request by j-j-gajjar**

### CODE REVIEW

**Comment by j-j-gajjar** [comment]
Hello @Ushie,
Because of the changes that I have made in the analysis file, this problem is being generated in the project, so I have tried to fix it.
Screenshot 2023-01-27 at 11 59 44 PM

**Review by Ushie — CHANGES_REQUESTED** [review]

File: lib/ui/views/app_selector/app_selector_view.dart [hunk 1]
```diff
@@ -19,7 +19,7 @@ class _AppSelectorViewState extends State<AppSelectorView> {
   @override
   Widget build(BuildContext context) {
     return ViewModelBuilder<AppSelectorViewModel>.reactive(
-      onModelReady: (model) => model.initialize(),
+      onViewModelReady: (model) => model.initialize(),
```
[the remark this answers is not in the document]
Comment (j-j-gajjar) [reply lib/ui/views/app_selector/app_selector_view.dart:22]: onModelReady' is deprecated and shouldn't be used. Prefer to use onViewModelReady instead.
Try replacing the use of the deprecated member with the replacement.

image

**Comment by j-j-gajjar** [comment]
@Ushie 
Please try to verify the Dart analyzer whenever a reviewer merge a PR.

Screenshot 2023-01-30 at 9 20 49 PM

**Comment by Ushie** [comment]
> @Ushie Please try to verify the Dart analyzer whenever a reviewer merge a PR.
> 
> Screenshot 2023-01-30 at 9 20 49 PM

Sorry about that! seems like I missed that one, I'll pay more attention next time

**Pull request merged**